### ISSUE TITLE
Test toolbox with AOIs in unusual coordinate systems

### ISSUE THREAD

**Issue by azvoleff** [ConservationInternational/trends.earth#86]
Can one of you test the toolbox using input AOIs with shapefiles with different coordinate systems? And preferable with shapefiles with more than one polygon (should just do an extent for the calculations)?

**Comment by mgroglich** [comment]
I am trying this for the Philippines with coordinates systems. I am running them using the public toolbox, do you want us to test this with a more current version? If so... the one in the file?

**Comment by mgroglich** [comment]
It seems to have only run for the first polygon. As for the projections, most of them give an error when downloading.

**Comment by azvoleff** [comment]
Yeah, it needs to be with the zipfile version of the plugin. For now ignore the option to set the output coordinate system (in the area screen in the new version). What is the error it gives you? Can you send the trends.earth log output and any message you get on the screen?
